Sources/KSCrashRecording/include/KSCrashReportWriter.h

```diff
@@ -241,9 +253,10 @@ typedef struct KSCrashReportWriter
     /** Internal contextual data for the writer */
     void* context;
 
-} KSCrashReportWriter;
+} NS_SWIFT_NAME(ReportWriter) KSCrashReportWriter;
 
-typedef void (*KSReportWriteCallback)(const KSCrashReportWriter* writer);
+typedef void (*KSReportWriteCallback)(const KSCrashReportWriter* writer)
+NS_SWIFT_UNAVAILABLE("Use Swift closures instead!");
```

Comment:
It actually resolves to closure. Generated interface in Xcode can't get types from imported headers. In Swift, there’s no need for a `typedef` for closures since it’s not a common practice in the language. 
<img width="507" alt="image" src="https://github.com/kstenerud/KSCrash/assets/23104281/87d73192-5dda-4979-93dc-8b8379cbb639">